### DISCUSSION TITLE
Welcome to dotfiles Discussions!

### DISCUSSION THREAD

**Discussion by FrostyNick** [FrostyNick/dotfiles#3] [Announcements]
## 👋 Welcome!
  I'm using Discussions as a place to chat about tech related stuff. You're welcome to:
  * Ask questions you’re wondering about.
  * Discuss ideas.
  * Judge or be creative.
  * Introduce yourself.

**Comment by ghost** [comment]
You should add examples on what dotfiles do such as adding screenshots in `readme.md`, etc.

**Reply by FrostyNick** [reply]
Updating the current screenshot is something I will prioritize when I have free time. Also, you're welcome to make GitHub issues for dotfiles examples and adding additional screenshots too if you want (ideally two separate issues). Otherwise I'll reference this in the issue tracker. Thanks for the feedback!

**Reply by FrostyNick** [reply]
Actually that's probably fine as one issue.

**Reply by FrostyNick** [reply]
For examples, I could make a video showcasing the dotfiles I've been thinking.
